doc/mgmt/SONiC_OC_Interface_HLD.md

```diff
@@ -392,6 +396,7 @@ sonic(conf-if-po4)# no switchport access Vlan
    mode - active<br>
    min-links - 0<br>
    fallback - disabled<br>
+   fast_rate - disabled<br>
 ```
 sonic(config)# interface PortChannel 1 mode active min-links 2 fallback
```

Comment:
Once the user specifies Min-links and fall back , he cannot change the values unless he deletes the port-channel and recreates it. Is there any other way to change these values without deleting the port-channel?

Reply:
This behavior was requested in 3.0, please discuss internally.